app/assets/js/graph.js

```diff
@@ -1,79 +1,70 @@
-import '../../../govuk/vendor/polyfills/Function/prototype/bind'
-import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
-import { nodeListForEach } from '../../../govuk/common'
 
-function Graph($module) {
-    this.$module = $module
-}
+var $module = document.querySelectorAll('[data-module="idsk-graph"]')[0];
```

Comment:
ak budes mat na stranke 2 grafy, tak to bude fungovat len pre ten prvy.. ale ak vo vzoroch vyuzivame len jeden tak to tak kludne mozme nechat